### DISCUSSION TITLE
修改 License

### DISCUSSION THREAD

**Discussion by FizzyElt** [FizzyElt/functional-programming#83] [General]
目前希望的調整成可引用，不可修改內容重新發布
不過我對這部份不太熟就是了

**Comment by dannypsnl** [comment]
不可重製會擋到其他人公開自己的筆記吧？

**Reply by FizzyElt** [reply]
可截取內容並附上來源呢？

**Reply by dannypsnl** [reply]
https://zh.wikipedia.org/wiki/%E7%9F%A5%E8%AF%86%E5%85%B1%E4%BA%AB

應該是可以研究一下 cc
